cfg/conf.sample.php

```diff
@@ -280,4 +280,5 @@ class = Filesystem
 ; Subresource integrity (SRI) hashes used in template files. Uncomment and set
 ; these for all js files used. See:
 ; https://github.com/PrivateBin/PrivateBin/wiki/FAQ#user-content-how-to-make-privatebin-work-when-i-have-changed-some-javascript-files
-;privatebin.js = sha512-[…]
+;privatebin.js = "sha512-[…]"
```

Comment:
I suspect that apart from the quoting, it requires the section header to get uncommented - I'm pretty sure this does work, because I use it often during development (setting it to empty string, so I don't need to update the hashes while I tinker). I'll retest to be sure, though.

Reply:
Nah, I did uncomment it, it did not work anyway...